<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-02T19:57:43 IST" name="Default suite" finished-at="2025-06-02T19:58:26 IST" duration-ms="42944">
    <groups>
    </groups>
    <test started-at="2025-06-02T19:57:43 IST" name="Default test" finished-at="2025-06-02T19:58:26 IST" duration-ms="42944">
      <class name="testNGAssertions.ToUnderstandSoftAssert">
        <test-method signature="verifyUsingSoftAssert()[pri:0, instance:testNGAssertions.ToUnderstandSoftAssert@49438269]" started-at="2025-06-02T19:57:43 IST" name="verifyUsingSoftAssert" finished-at="2025-06-02T19:58:26 IST" duration-ms="42894" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Electronics Page Is Not Displayed expected [Demo Web Shop. Electronics ] but found [Demo Web Shop. Electronics],
Cell Phone Page is Not Displayed expected [Demo Web Shop. Cell phones ] but found [Demo Web Shop. Cell phones]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Electronics Page Is Not Displayed expected [Demo Web Shop. Electronics ] but found [Demo Web Shop. Electronics],
Cell Phone Page is Not Displayed expected [Demo Web Shop. Cell phones ] but found [Demo Web Shop. Cell phones]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testNGAssertions.ToUnderstandSoftAssert.verifyUsingSoftAssert(ToUnderstandSoftAssert.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsingSoftAssert -->
      </class> <!-- testNGAssertions.ToUnderstandSoftAssert -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
